python/src/LOLAVoronoi_doc.i.in

```diff
@@ -0,0 +1,150 @@
+%feature("docstring") OT::LOLAVoronoi
+"LOLA-Voronoi sequential design algorithm.
+
+This class implements the sequential design algorithm described in [crombecq2011]_ which allows one
+to sequentially generate new input samples to an existing design built from input/output samples.
+It combines an exploration criterion based on Voronoi tesselation to identify
+undersampled input design regions and an exploitation criterion based on a measure
+of the nonlinearity of the response surface.
+
+Input sample Voronoi cell relative volumes are estimated by Monte Carlo:
+
+.. math::
+
+    V(\vect{p}_i) = \int_{\vect{x} \in \Rset^d} 1_{i = \argmin_{j \in [1, N]} ||\vect{p}_j - \vect{x}||_2} dx
```

Comment:
p(x)

Reply:
An upper case letter should be chosen only for a random variable or a matrix.

```suggestion
    v(\vect{p}_i) = \int_{\vect{x} \in \Rset^d} 1_{i = \argmin_{j \in [1, N]} ||\vect{p}_j - \vect{x}||_2} dx
```